gooddata-sdk/gooddata_sdk/catalog/workspace/model_container.py

```diff
@@ -97,36 +117,49 @@ def find_label_attribute(self, id_obj: IdObjType) -> Union[CatalogAttribute, Non
         return None
 
     def _valid_objects(self, ctx: ValidObjectsInputType) -> ValidObjects:
-        return self._valid_obf_fun(ctx)
+        if self._valid_obj_fun:
+            return self._valid_obj_fun(ctx)
+        else:
+            return {}
```

Comment:
Return type should be ValidObjects and you return empty dictionary. Is it intended?